### PR TITLE
Address review feedback for core PR

### PR DESCRIPTION
**Pull request by mukeshpanchal27**
This PR address review feedback for core PR.

### CODE REVIEW

File: .github/workflows/performance.yml
Comment (mukeshpanchal27) [inline .github/workflows/performance.yml]:
```suggestion
```

Once all the reviewer feedback has been addressed, we will remove this line.

File: .github/workflows/performance.yml
Comment (joemcgill) [inline .github/workflows/performance.yml]: We can probably simplify these to:

```suggestion
      - '[6-9]+.[2-9]'
      - '[6-9]+.[2-9].[0-9]+'
```

File: .github/workflows/performance.yml
Comment (joemcgill) [reply .github/workflows/performance.yml]: Feel free to remove this before merging this PR and make future updates directly in the feature branch. No need to get separate reviews on changes to the open PR on core.

**Review by joemcgill — APPROVED** [review]
Couple of suggestions, but feel free to merge this once those are taken care of. I don't think we need to do additions to the Core PR as separate PRs.

**Pull request merged**
